grpc/v1/webserver.proto

```diff
@@ -380,7 +380,9 @@ message FileMatch {
   // other packages can print some diagnostics.
   string debug = 2;
 
-  string file_name = 3;
+  // The repository-relative path to the file.
+  // 🚨 Warning: file_name might not be a valid UTF-8 string.
+  bytes file_name = 3;
```

Comment:
Note: This change doesn't break the protobuf binary protocol. See https://protobuf.dev/programming-guides/proto3/#updating

> string and bytes are compatible as long as the bytes are valid UTF-8.